lambdas/death-minimisation/src/test/resources/uk/gov/di/data/lep/approvals/DeathMinimisationTest.minimsationSnapshotTest.approved.txt

```diff
@@ -0,0 +1 @@
+{"aud":null,"events":{"https://ssf.account.gov.uk/v1/deathRegistrationUpdate":{"deathRegistrationID":123456,"deathRegistrationUpdateReason":null,"recordUpdateTime":{"value":"2022-11-02T12:00:00.000"},"subject":{}}},"exp":null,"iat":null,"iss":null,"jti":null,"nbf":null,"sub":null,"toe":null,"txn":null}
```

Comment:
Do we know why iat and txn are null here?

Reply:
No, I can have a look, my _guess_ would be our builder is setting them to null rather than setting something valid

Reply:
Having checked in more detail, we just don't set them in the test data for this case - I can go and set them if we think that's valuable though